src/treq/client.py

```diff
@@ -36,16 +34,22 @@
 from requests.cookies import cookiejar_from_dict, merge_cookies
 
 if _PY3:
-    from urllib.parse import urlunparse, urlencode as _urlencode
+    from urllib.parse import urlencode as _urlencode
 
     def urlencode(query, doseq):
         return _urlencode(query, doseq).encode('ascii')
     from http.cookiejar import CookieJar
 else:
     from cookielib import CookieJar
-    from urlparse import urlunparse
     from urllib import urlencode
 
+try:
+    # The old location was quixotically deprecated and might actually be
```

Comment:
"quixotically" feels like maybe a *little* bit of unnecessary editorializing ;-).  Do you have a link you want to drop in here?

Reply:
I have... strong feelings here.

I do not think that deprecating this import alias is a meaningful improvement to the Python standard library. This alias could be left in place, undocumented, forever, at little cost.

I have fixed this deprecation warning in countless places since it was introduced, both publicly and privately. I'm really quite tired of it. It's makework that doesn't deliver value to anyone.

I feel that this change is frankly _disrespectful_ to library maintainers. I mean, here are some CPython devs trying to get the html5lib maintainer to do a release _so that they don't break his code_: https://github.com/html5lib/html5lib-python/issues/419#issuecomment-591152347 This is not a good look.

And then, well, they eventually decide not to do it yet. Note Victor's response:

> FYI [removal of] aliases to ABC in collections have been reverted in Python 3.9, but they will be removed again in Python 3.10: https://bugs.python.org/issue39674